packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/UpdatePalletFrozenStatus/UpdatePalletFrozenStatus.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { useFormContext } from 'react-hook-form'
+
+import { InputComponent } from '@/common/components/forms'
+import { Row } from '@/common/components/Modal'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { TextMedium } from '@/common/components/typography'
+
+import { SelectPalletFrozenStatus } from './SelectPalletFrozenStatus'
+
+export type PalletFrozenStatus = 'Enabled' | 'Disabled'
+
+export const UpdatePalletFrozenStatus = () => {
+  const { watch, setValue } = useFormContext()
+
+  const setPalletFrozenStatus = (selected: PalletFrozenStatus) => {
+    if (selected === 'Enabled') {
+      setValue('updatePalletFrozenStatus.frozen', true, { shouldValidate: true })
```

Comment:
So based on [this](https://github.com/Joystream/joystream/blob/nara/tests/network-tests/src/flows/proposals/index.ts#L161):
![image](https://github.com/Joystream/pioneer/assets/6571453/2761f679-bee4-4318-ad27-6058b1c46583)

"Enabled" should mean frozen to `false` and "Disabled" frozen to `true`

Since a toggle should used instead of a select (see: https://github.com/Joystream/pioneer/pull/4625#discussion_r1389241248) this isn't so relevant anymore but keep this in mind when labeling the toggle. It's kinda counterintuitive.